src/app/pyramid-visualization/pyramid-visualization-new/pyramid-visualization-new.component.html

```diff
@@ -1,28 +1,38 @@
-<div class="modal-header">
-  <h4 class="modal-title">
-    <span class="glyphicon glyphicon-star"></span> Visualization name
-  </h4>
-</div>
-<div class="modal-body">
-  <form name="visualizationDialog">
-    <div class="form-group">
-      <label class="control-label" for="name">Name:</label>
-      <input type="text" class="form-control" name="name" id="name"
-             [(ngModel)]="visualization.name"
-             autofocus
-             required>
-      <small class="form-text text-muted">Enter the visualization name.</small>
-    </div>
-    <mat-checkbox [(ngModel)]="visualization.publiclyShared" name="true">Public visualization</mat-checkbox>
-    <br *ngIf="!visualization.publiclyShared">
-    <span class="text-danger" *ngIf="!visualization.publiclyShared">This visualization will only be available to you.</span><br>
-  </form>
-</div>
-<div class="modal-footer">
-  <button type="button" class="btn btn-danger" (click)="cancel()">Cancel</button>
-  <button type="button" class="btn btn-success" (click)="save()"
-          [disabled]="">
-    Save
-  </button>
-</div>
+<form>
+  <div class="flex flex-column gap-2">
+    <label for="name" class="font-semibold w-6rem">Name</label>
+    <input pInputText [(ngModel)]="visualization.name" id="name" name="name" class="flex-auto" autocomplete="off" />
+    <span class="p-text-secondary block mb-5">Enter the visualization name.</span>
+  </div>
+  <div class="flex justify-content-end gap-2">
+    <p-button label="Cancel" severity="secondary" (onClick)="cancel()" />
+    <p-button label="Save" (onClick)="save()" [disabled]="!visualization.name" type="submit"/>
+  </div>
+</form>
+<p-toast position="top-center"/>
+
+<!--<div class="modal-header">-->
+<!--  <h4 class="modal-title">-->
+<!--    <span class="glyphicon glyphicon-star"></span> Visualization name-->
+<!--  </h4>-->
+<!--</div>-->
+<!--<div class="modal-body">-->
+<!--  <form name="visualizationDialog">-->
+<!--    <div class="form-group">-->
+<!--      <label class="control-label" for="name">Name:</label>-->
+<!--      <input type="text" class="form-control" name="name" id="name"-->
+<!--             [(ngModel)]="visualization.name"-->
+<!--             autofocus-->
+<!--             required>-->
+<!--      <small class="form-text text-muted">Enter the visualization name.</small>-->
+<!--    </div>-->
+<!--  </form>-->
+<!--</div>-->
+<!--<div class="modal-footer">-->
+<!--  <button type="button" class="btn btn-danger" (click)="cancel()">Cancel</button>-->
+<!--  <button type="button" class="btn btn-success" (click)="save()"-->
+<!--          [disabled]="">-->
+<!--    Save-->
+<!--  </button>-->
+<!--</div>-->
```

Comment:
L14 to L37: Should this commented code remain?

Reply:
Fixed